src/index.js

```diff
@@ -4,8 +4,8 @@ import wrappers from './wrappers';
 const debug = makeDebug('feathers-rest');
 
 function formatter(req, res, next) {
-  if(!res.data) {
-    next();
+  if(typeof res.data === 'undefined') {
```

Comment:
As long as it works it's fine.  I just wanted to see if you had some kind of ninja tricks that I haven't learned yet